spec/miscellaneous/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -97,3 +97,81 @@
     end
   end
 end
+
+describe "rake After:add_user_id_to_kudos" do
```

Comment:
I hate these tests. 😭 The old ones were neater, but didn't make much sense with the new approach. I will gladly accept advice on how to rewrite these from the ground up.

Reply:
How so? The old tests should still pass, however the rake chooses to do things.

Reply:
They did still pass, but since we're going through the users table rather directly working on the kudos table now, it didn't make sense to me to check that guest kudos were going untouched... but it did seem to be important to test a case where the user had multiple pseuds with kudos, which wasn't a consideration before.

Reply:
:+1: